<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormTelaPrincipal</class>
 <widget class="QMainWindow" name="FormTelaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tela Principal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>201</width>
      <height>361</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::NoFocus</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelNomeUsuario">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Plain</enum>
       </property>
       <property name="text">
        <string>Adminstrador</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonInicio">
       <property name="text">
        <string>Inicio</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonEstoque">
       <property name="text">
        <string>Estoque</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonGerenciarUsuarios">
       <property name="text">
        <string>Gerenciar Usu√°rios</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonEditarPerfil">
       <property name="text">
        <string>Editar Perfil</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>591</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_1Inicio">
     <widget class="QLabel" name="labelBemVindo">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>371</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Bem Vindo !</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelNomeCargo_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>280</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Administrador</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2Financeiro"/>
    <widget class="QWidget" name="page_3Estoque">
     <widget class="QTableWidget" name="tableWidgetComida">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>531</width>
        <height>171</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>TIRA GOSTO</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>VALOR</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>QUANTIDADE</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="labelEstoque">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Estoque</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidgetBebida">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>531</width>
        <height>171</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>BEBIDAS</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>VALOR</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>QUANTIDADE</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="labelNomeItem">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>430</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nome do Item</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditNomeItem">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>450</y>
        <width>571</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelValorItem">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>480</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Valor</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_9">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>470</y>
        <width>191</width>
        <height>64</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="labelTipoItem">
         <property name="text">
          <string>Tipo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxItem">
         <item>
          <property name="text">
           <string>Selecione...</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tira Gosto</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Bebida</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLineEdit" name="lineEditValorItem">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>500</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelQuantidadeItem">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>480</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Quantidade</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditQuantidadeItem">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>500</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelCadastrarItem">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>400</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Cadastrar Item</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAdicionarItem">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>540</y>
        <width>281</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>ADICIONAR ITEM</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCancelarItem">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>540</y>
        <width>271</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>CANCELAR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRemoverBebida">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>230</y>
        <width>31</width>
        <height>171</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="QIcon::ThemeIcon::EditDelete"/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRemoverComida">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>50</y>
        <width>31</width>
        <height>171</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="QIcon::ThemeIcon::EditDelete"/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4GerenciarUsuarios">
     <widget class="QTableWidget" name="tableWidgetUsuarios">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>561</width>
        <height>381</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>NOME</string>
       </property>
       <property name="font">
        <font>
         <kerning>true</kerning>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>EMAIL</string>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>CARGO</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="labelGerenciarUsuarios">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Gerenciar Usu√°rios</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_10">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>520</y>
        <width>551</width>
        <height>44</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButtonCadastrar">
         <property name="text">
          <string>Cadastrar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonEditar">
         <property name="text">
          <string>Editar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonExcluir">
         <property name="text">
          <string>Excluir</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5EditarPerfil">
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>139</y>
        <width>521</width>
        <height>421</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="labelNome">
               <property name="text">
                <string>Nome</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_4">
               <property name="frameShape">
                <enum>QFrame::Shape::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Shadow::Raised</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditNome"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_6">
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="labelSobrenome">
               <property name="text">
                <string>Sobrenome</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditSobrenome"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_8">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="labelEmail">
            <property name="text">
             <string>Email</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditEmail"/>
          </item>
          <item>
           <widget class="QLabel" name="labelSenha">
            <property name="text">
             <string>Senha</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditSenha">
            <property name="echoMode">
             <enum>QLineEdit::EchoMode::Password</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelNovaSenha">
            <property name="text">
             <string>Nova senha</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditNovaSenha">
            <property name="echoMode">
             <enum>QLineEdit::EchoMode::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_7">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButtonSalvar">
            <property name="text">
             <string>Salvar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonCancelar">
            <property name="text">
             <string>Cancelar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="labelEditarPerfil">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Editar Perfil</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelNomeUsuarioEditar">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>70</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Dilma Roussef</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelNomeCargoEditar">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>100</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Administrador</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonSair">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>550</y>
      <width>183</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>SAIR</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
